app/TracingStrategyAssets.ts

```diff
@@ -91,6 +93,53 @@ export function useAssets(): Record<string, string> {
     ? t('label.home_no_contact_subtext')
     : t('label.home_no_contact_subtext_bluetooth');
 
+  // Export Pages
+
+  const exportExitRoute = isGPS ? 'ExportStart' : 'SettingsScreen';
```

Comment:
What do you think about moving the screen names outside of the tracingStrategy assets ?

Reply:
This was a bit weird bc of the flow that is in GPS, which I think is changing. On the GPS side, the feature flag puts this on the tab navigation. I am hoping that we can coordinate with design to have a more similar flow and not have to do this. We have put these routes in here in the past as an intermediate and we usually end up being able to get rid of them once design for GPS and BT catch up.

I can move them to a different hook or something if we see this pattern staying in place.